camp/jbolotski/beam_break_motor/src/main/java/frc/robot/common/Types.java

```diff
@@ -0,0 +1,11 @@
+package frc.robot.common;
+
+/**
+ * Common data types
+ */
+public class Types {
```

Comment:
No class definition needed for enums.  Likely can just have the enum defined.  Once you do that, however I think java likes to have the enum name as the .java file name so that will trigger file-rename.

Reply:
I like to have all the robot types in one class so that you can import them all together.  It's akin to what we do for Constants.java.  Granted, there is just one type for this mini-project, I expect there to be more types in a real robot.  What do you think?

Reply:
So standard practice at MS is we put enums that are public in their own file (but we roll C#).  As for Java I just looked on the internet:

https://softwareengineering.stackexchange.com/questions/164884/best-practice-for-packing-java-enums
https://stackoverflow.com/questions/14431472/why-is-an-enum-declared-in-a-separate-file-in-java

I think 'findability' is the main benefit.  It can be a pain if we only have 3 for a project.

Reply:
Thanks!